tests/function/test_config.py

```diff
@@ -88,7 +88,13 @@ def test_interval_in_virtwho_conf(self, virtwho, globalconf):
             rhsm_log = virtwho.rhsm_log_get()
             assert "No data to send, waiting for next interval" in rhsm_log
         else:
-            assert result["loop"] in [60, 61]
+            assert result["loop"] in [60, 61, 62, 63]
+
+        globalconf.update("global", "interval", "120")
+        result = virtwho.run_service()
+        assert (
+            result["send"] == 1 and result["error"] == 0 and result["interval"] == 120
+        )
```

Comment:
add the interval=120 step, which is defined in the docstring but missed in the steps.

Reply:
Actually, we have already discussed this. Since the behavior should be the same for `interval == 60`, we removed it (forgot to update the docstring). However, if you believe it is necessary to test this, we can include it.

Reply:
yes, removed it and the docstring.